lib/v2/douban/other/doulist.js

```diff
@@ -56,7 +58,7 @@ module.exports = async (ctx) => {
 
     ctx.state.data = {
         title,
-        link,
+        url,
```

Comment:
https://docs.rsshub.app/joinus/advanced/advanced-feed
There's no `url` property.

Reply:
sry,I'm a freshman for rss.